src/components/publishing/article.tsx

```diff
@@ -6,9 +6,13 @@ import FeatureLayout from "./layouts/feature_layout"
 import StandardLayout from "./layouts/standard_layout"
 import FullscreenViewer from "./sections/fullscreen_viewer/fullscreen_viewer"
 import Sections from "./sections/sections"
+import { Layout } from "./typings"
 
-interface ArticleProps {
-  article: any
+export interface ArticleProps {
+  article: {
+    layout: Layout
+    [x: string]: any
```

Comment:
Just making note of this change to get our Layout type working across components. Not sure if there's a cleaner solution so lmk if you can think of one.

If you see the next file below in `fixtures/articles.ts`, we reuse this interface when defining the various types of articles. This makes testing a little more sane too since it gives proper names to articles instead of pulling them arbitrarily from an array. Soo the diff grew by a lot but it's mostly whitespace.

Reply:
Is the interface for an ‘article’ always going to be the same? If so, then maybe you could move the interface declaration for it to the central typings too and import/use it here like so:

```ts
import { ArticleData } from "./typings"

interface ArticleProps {
  article: ArticleData
}
```

Then in the `fixtures/articles.ts` file you won’t need to do `ArticleProps["article"]` but just `ArticleProps`, which makes more sense.

I didn’t entirely follow what your question about a better way was about, does this answer it?

Reply:
oh nice! yeah that seems more flexible. I was moreso wondering about the `[x: string]: any` part since we needed to pull out layout to define it.